pkg/cli/root.go

```diff
@@ -223,6 +223,14 @@ func validateConfiguration(config *config.Config) error {
 		}
 	}
 
+	if config.HTTP.ReadOnly {
+		log.Warn().Msgf("ReadOnly option is deprecated, use AccessControl instead")
```

Comment:
Deprecated means we are still supporting it, but we will not in the future.
In this PR we remove it, so I don't think it's the right wording.

Do we want to remove or deprecate it? 
Deprecate could be better for existing users of this setting if there are any, but would take some extra maintenance.